packages/design-system/src/components/FormLabel/useFormLabel.tsx

```diff
@@ -92,13 +92,13 @@ export interface UseFormLabelProps extends FormFieldProps {
 export function useFormLabel<T extends UseFormLabelProps>(props: T) {
   // TODO: Once we're on React 18, we can use the `useId` hook
   const generatedId = useRef(uniqueId('field_')).current;
-  const id = props.id ?? generatedId;
-  const labelId = props.labelId ?? `${id}-label`;
-  const errorId = props.errorId ?? `${id}-error`;
 
   const {
     className,
     label,
+    id = props.id ?? generatedId,
+    labelId = props.labelId ?? `${id}-label`,
+    errorId = props.errorId ?? `${id}-error`,
```

Comment:
Sorry, I didn't mean in this hook, because I think these are relied on elsewhere. I meant pulling the props out of `...selectProps` that you don't want to end up in `fieldProps`, because all extra props that are passed to `useFormLabel` get placed into `fieldProps`. Most of everything that is in `fieldProps` right now in `Dropdown`—as far as I can tell—are things that you want applied to the `select` element, so you'd just need to pull out the things that don't belong like `errorMessage`

Reply:
From https://github.com/CMSgov/design-system/pull/2243#discussion_r1019657599

Reply:
Example: https://github.com/CMSgov/design-system/pull/2243/files#diff-9e9a5e7a70e71f7cbf1f7afbd4da0f819174272f4d4d516f80cb4f112fcb7871R124